Loop/Managers/DosingStrategy.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  DosingStrategy.swift
+//  Loop
+//
+//  Created by Jonas Björkert on 2023-06-03.
+//  Copyright © 2023 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import LoopKit
+import LoopCore
+
+protocol DosingStrategy {
```

Comment:
"Dosing Strategy" is a term already used to describe temp basal vs automatic bolus.  Maybe call this `ApplicationFactorStrategy`?